Makefile

```diff
@@ -65,8 +70,3 @@ functional-test:
 .PHONY: bench
 bench: go-generate
 	go test ./... -bench=. -run=XXX -benchmem
-
-.PHONY: initialize-db
-initialize-db: go-generate
```

Comment:
As far as I can tell this is no longer needed.

Reply:
because the migrate command creates the tables, no?

Reply:
Yep. I don't think we used this for anything else right?

Reply:
No